tests/test_smart_task.py

```diff
@@ -55,6 +55,18 @@
     "9044": [{"9046": 8, "9047": 15}],
 }
 
+TASK_OPTIONAL_DIMMER = {
+    "9001": "Light and dark",
+    "9002": 1613335145,
+    "9003": 318615,
+    "5850": 1,
+    "9040": 2,
+    "9041": 127,
+    "9042": {"5850": 1, "15013": [{"9003": 65553}]},
+    "9043": {"5850": 0, "15013": [{"9003": 65553}]},
```

Comment:
This is the response captured in the issue. Do we know what 9043 represents? We don't have that one in our `const` module.

Reply:
I'm unable to test using the app for the next few weeks unfortunately, I suspect it relates to controlling an outlet via smart task. Is there a way I can confirm this somehow, given my setup?